### PR TITLE
feat: Reorganize x/oracle

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

Moves the oracle's existing files around a bit, adds a `module.go` & `genesis.go` file, and a client/cli

Progress on: #250

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added appropriate labels to the PR
- [x] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/oracle/client/cli/query.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/spf13/cobra"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(queryRoute string) *cobra.Command {
+	// Group oracle queries under a subcommand
```
Comment (alexanderbez) [inline x/oracle/client/cli/query.go:13]:
```suggestion
```

File: x/oracle/client/cli/tx.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+var (
+	DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
```
Comment (alexanderbez) [inline x/oracle/client/cli/tx.go:14]: What's this? Is it used?

File: x/oracle/genesis.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/x/oracle/keeper"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// InitGenesis initializes the x/oracle module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	k.SetParams(ctx, genState.Params)
```
Comment (alexanderbez) [inline x/oracle/genesis.go:13]: Is there nothing else to do when initializing genesis?

**Review by alexanderbez — APPROVED** [review]

File: x/oracle/client/cli/tx.go (hunk 2 above)
Comment (adamewozniak) [reply x/oracle/client/cli/tx.go:14]: Nope, got this from a starport tutorial project - I'll take it out

File: x/oracle/genesis.go (hunk 3 above)
Comment (adamewozniak) [reply x/oracle/genesis.go:13]: There's more to do here & `ExportGenesis`, just wanted to add it after I started pruning the stuff we don't need - I can open an issue for it

**Comment by alexanderbez** [comment]
Feel free to squash merge this when ready 👍

**Pull request merged**